src/Serilog.Sinks.RollingFile/Sinks/RollingFile/TemplatedPathRoller.cs

```diff
@@ -29,18 +29,42 @@ class TemplatedPathRoller
         const string OldStyleDateSpecifier = "{0}";
         const string DateSpecifier = "{Date}";
         const string DateFormat = "yyyyMMdd";
+        const string HourSpecifier = "{Hour}";
+        const string HourFormat = "yyyyMMddHH";
+        const string HalfHourSpecifier = "{HalfHour}";
+        const string HalfHourFormat = "yyyyMMddHHmm";
         const string DefaultSeparator = "-";
 
+        const string MatcherMarkSpecifier = "date";
```

Comment:
Good idea pulling these out as constants :+1:

Just on the naming, could we perhaps use:

```csharp
const string SpecifierMatchGroup = "specifier";
const string SequenceNumberMatchGroup" = "sequence";
```